app/src/main/java/org/sopt/soptseminar/login/LoginFragment.kt

```diff
@@ -0,0 +1,67 @@
+package org.sopt.soptseminar.login
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.Navigation
+import androidx.navigation.fragment.findNavController
+import org.sopt.soptseminar.R
+import org.sopt.soptseminar.databinding.FragmentLoginBinding
+
+class LoginFragment : Fragment() {
+    private lateinit var binding: FragmentLoginBinding
+    private lateinit var viewModel : LoginViewModel
```

Comment:
```suggestion
    private val viewModel: LoginViewModel by viewModels()
```

Android KTX를 사용하면 뷰모델 위임 초기화로 더 깔끔하게 생성할 수 있습니다.

Reply:
여얼 알겠슴다

Reply:
현우형이 말해준 부분에서 위임 초기화를 하는 이유도 한번 알아보면 좋을 것 같다는 생각이 들어!